awx/main/utils/external_logging.py

```diff
@@ -57,7 +57,11 @@ def construct_rsyslog_conf_template(settings=settings):
             'action.resumeRetryCount="-1"',
             'template="awx"',
             'errorfile="/var/log/tower/rsyslog.err"',
-            f'action.resumeInterval="{timeout}"'
+            f'action.resumeInterval="{timeout}"',
+            'queue.filename="awx-external-log"',
+            'queue.size="100000"',
```

Comment:
https://rsyslog-doc.readthedocs.io/en/latest/concepts/queues.html#limiting-the-queue-size